src/pages/OfferNew/Perils/PerilModal.tsx

```diff
@@ -351,7 +352,7 @@ export const PerilModal: React.FC<PerilModalProps & ModalProps> = (props) => {
                   props.setCurrentPeril(index % tripledPerils.length)
                 }
               >
-                {peril.icon}
+                <img src={getIconUrl(peril.icon.variants.light.svgUrl)} />
```

Comment:
I had this idea that maybe we could use ajax calls to fetch the icons' contents, for example if we wanted to `fill`/tint them. But maybe they're in the right color already?

Reply:
They are black, and they should be in the slightly grayer variant. So that solution can be good to add but I'm thinking in a separate PR to just get the other parts in place or WDYT?

Reply:
Good thinking 👍 i like taking on this type of debt more than other kinds of shortcuts - it's extremely visible and will be easy to detect in a qa session, meaing we'll _actually_ fix it "later"